third_party/xla_client/pjrt_computation_client.cc

```diff
@@ -332,6 +354,8 @@ std::vector<ComputationClient::ComputationPtr> PjRtComputationClient::Compile(
     if (instance.is_sharded) {
       // TODO(yeounoh) multi-host, multi-slice configurations
       compile_options.executable_build_options.set_use_spmd_partitioning(true);
+      compile_options.executable_build_options
+          .set_allow_spmd_sharding_propagation_to_output({false});
```

Comment:
can you add a comment here to explain why it is needed?

Reply:
Ack.

Reply:
Did I miss the comment somewhere?

Reply:
Done